be/src/runtime/datetime_value.h

```diff
@@ -435,7 +436,8 @@ class DateTimeValue {
     }
 
     int64_t second_diff(const DateTimeValue& rhs) const {
-        return unix_timestamp() - rhs.unix_timestamp();
+        return unix_timestamp(TimezoneDatabase::default_time_zone) 
+            - rhs.unix_timestamp(TimezoneDatabase::default_time_zone);
```

Comment:
For second_diff which is a literal compare, I think we should not introduce timezone. Which can bring some cost which isn't needed. Such as search if timezone exists.

And I think we can introduce another seconds such as seconds from 0000-01-01 00:00:00, which has no relation with time zone.